examples/todo-with-auth0/apps/admin/.env.development

```
@@ -1,7 +1,18 @@
+<<<<<<< CURRENT
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The <<<<<<< CURRENT key should be with a value or have an equal sign

(KeyWithoutValue)